CHANGELOG.rst

```diff
@@ -1,5 +1,38 @@
 Changelog
 =========
+1.5.1.dev0 (2020-01-28)
+-----------------------
+MLflow 1.5.1.dev0 includes several major features and improvements:
+
+Features:
+
+- File store optimizations (#2339, @jonas)
+- Add feature importance plot for the XGBoost flavor (#2337, @harupy)
+- New runs table column view (#2251, @Zangr)
+- Add EarlyStopping integration to TensorFlow.Keras autologging (#2301, @juntai-zheng)
+- EarlyStopping Callback support for Keras Autologging (#2219, @juntai-zheng)
+- Add autodetection of job environments to R client (#2272, @dbczumar)
+- Add autodetection of job environments to Java client (#2286, @dbczumar)
+- Add auto logging functionality for LightGBM flavor. (#2275, @harupy)
```

Comment:
auto logging  -> autologging
Combine two autologging features: LightGBM and XGBoost